### ISSUE TITLE
Document in the contributing guidelines the Cardano design principles they have to adhere to.

### ISSUE THREAD

**Issue by dnadales** [input-output-hk/ouroboros-consensus#407]
We should explain these both to our team and for external contributors.

This issue becomes particularly relevant now that Cardano development is moving to a member's based organization. Contributors should know in advance which design principles they must there to.

**Comment by nfrisby** [comment]
Agreed, very important. For us specifically, it would likely be useful to specialize those principles to our code base and perhaps to add some new principles specific to our code base.
